### ISSUE TITLE
65: Players drop weapons

### ISSUE THREAD

**Issue by theonlypwner** [acreloaded/acr#83]
**Players drop weapons** by `tbcc_1@hotmail.com`
Issue 65 posted to Google Code on 2012 Oct 24 at 04:12:57

Current status: Acknowledged
Current labels: `Type-Enhancement`, `Priority-High`, `Component-UI`, `Component-Engine`, `Component-Logic`, `Component-Net`, `Effect-Gameplay`, `Effect-Protocol`

`tbcc_1@hotmail.com` on 2012 Oct 24 at 04:12:57:
> When player is killed, he/she drops the weapon that he/she was using before being killed

`harryspag10` on 2012 Oct 24 at 11:19:42:
> Would be good but there are not enough weapons in the game for this to have a paint or make sense... So first he should make new weapons (as said in another issue)!

`harryspag10` on 2012 Oct 24 at 11:20:10:
> Then there would be a point to picking up weapons!

`theonlypwner` on 2012 Oct 24 at 22:20:56:
> We just don't have any good way to render weapons in the world (world weapons are attached to a player, except the knife/nade throwables).

> **Labels**: -Priority-Unset Priority-High Component-UI Component-Engine Component-Logic Component-Net Effect-Gameplay Effect-Protocol
> **Status**: Acknowledged

`theonlypwner` on 2012 Oct 24 at 22:30:55:
> They can still drop ammo, and then your weapon can be refilled. But first, more weapons should be done.

> **Blocked by**: 50

`www.jon99` on 2012 Oct 27 at 00:40:38:
> ammo could be an option, thanks for your replies!

**Comment by theonlypwner** [comment]
Issue 50 = #68
